hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/StandardCryptoService.java

```diff
@@ -15,18 +15,23 @@
  */
 package com.hedera.node.app.service.token.impl;
 
+import com.hedera.node.app.service.mono.token.impl.AccountStore;
 import com.hedera.node.app.service.token.CryptoPreTransactionHandler;
 import com.hedera.node.app.service.token.CryptoService;
 import com.hedera.node.app.spi.PreHandleContext;
 import com.hedera.node.app.spi.state.States;
 import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Objects;
 
-/** Standard implementation of the {@link CryptoService} {@link com.hedera.node.app.spi.Service}. */
+/** An implementation of the {@link CryptoService} interface. */
```

Comment:
We had a discussion some time ago that it won't be the best idea to call the implementations of our services `*Impl` since we could have more than 1 implementation. Based on that I decided to use `Standard*` as a name definition for the service implementations. I deleted the old dummy and renamed the `CryptoServiceImpl` to `StandardCryptoService`.